plugins/catalog-backend/src/next/NextCatalogBuilder.ts

```diff
@@ -252,7 +263,11 @@ export class NextCatalogBuilder {
 
     const db = new CommonDatabase(dbClient, logger);
 
-    const processingDatabase = new DefaultProcessingDatabase(dbClient, logger);
+    const processingDatabase = new DefaultProcessingDatabase(
+      dbClient,
+      logger,
+      this.refreshIntervalSeconds,
```

Comment:
reaching a point, i guess, where we want to start using an object instead of multiple parameters

Reply:
fixed